spec/api/comment_thread_spec.rb

```diff
@@ -477,6 +477,18 @@ def test_unicode_data(text)
         check_thread_result_json(nil, thread, parsed)
       end
 
+      context 'when requesting the thread for informational purposes' do
+        subject do
+          get "/api/v1/threads/#{thread.id}", with_responses: false # we're asking for no responses here.
```

Comment:
Yeah, it's hard because what I want is for the default to be `with_responses: false` so that you have to opt-in to getting the responses.  The only way to make it work in a backwards-compatible way, though, I think, is to add this support but default to always returning responses, then make the LMS change where it specifically requests responses when rendering a single thread, and then finally making the default to be `with_responses: false` in the forums.

Open to other suggestions, though.  The tests definitely fail -- like some of them will fail for sure -- if no children are present, they're just not neatly bundled next to this specific one.